abel/tools/symmetry.py

```diff
@@ -165,8 +165,8 @@ def put_image_quadrants (Q, odd_size=True, vertical_symmetry=False,
         Bottom = np.flipud(np.concatenate((np.fliplr(Q2), Q3), axis=1))
     else:
         # odd size image remove extra row/column added in get_image_quadrant()
-        Top    = np.concatenate((np.fliplr(Q1[:-1,:-1]), Q0[:-1,:]), axis=1)
-        Bottom = np.flipud(np.concatenate((np.fliplr(Q2[:,:-1]), Q3), axis=1))
+        Top    = np.concatenate((np.fliplr(Q[1])[:-1,:-1], Q[0][:-1,:]), axis=1)
```

Comment:
Can you confirm that this change is correct? Seems to revert back to the original image before symmetrization was applied. Although maybe the "put_image_quadrants" function shouldn't actually care about vertical and horizontal symmetry, since this was supposedly taken care of by the "get_image_quadrants" function...